e2e/app/page/workspace-analysis-page.ts

```diff
@@ -143,7 +125,7 @@ export default class WorkspaceAnalysisPage extends WorkspaceBase {
   async copyNotebookToWorkspace(notebookName: string, destinationWorkspace: string, destinationNotebookName?: string): Promise<void> {
     // Open Copy modal.s
     const resourceCard = new DataResourceCard(this.page);
-    const notebookCard = await resourceCard.findCard(notebookName, CardType.Notebook);
+    const notebookCard = await resourceCard.findCard(notebookName, ResourceCard.Notebook);
```

Comment:
There can also be a utils method that takes in this.page , resource name and card type.
const resourceCard = new DataResourceCard(this.page); can move in that util method